### ISSUE TITLE
OpenAPI: Add support for const

### ISSUE THREAD

**Issue by JonasHeylen** [softwaremill/sttp-apispec#56]
The `const` keyword that is part of the JSON Schema spec (https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-6.1.3) is not yet supported by sttp-apispec.

Adding support for this would make it possible to define enums with a description of each enum value, as suggested in https://github.com/json-schema-org/json-schema-vocabularies/issues/47 and https://github.com/OAI/OpenAPI-Specification/issues/348.

Example:
```yaml
  oneOf:
    - const: D
      description: Suspicious transaction
    - const: I
      description: Failed data validation check
```

**Comment by hamnis** [comment]
This can be closed, was fixed in  #69

**Comment by adamw** [comment]
Thanks, closing :)
